packages/terra-list/docs/ListSection.md

```diff
@@ -0,0 +1,30 @@
+# List Section
+
+The list section component provides the ability to display section headers and collapse a section within a logical group. Through the use of React fragments the end result of the render method is still a flat list. With a flat list we are able to address many accessibility concerns associated with nested list elements.
```

Comment:
IMO most of the stuff about fragments could be removed from these docs. Not that it's not interesting, it's just overkill for consumer documentation.

I'd also rephrase the stuff below to say "The section cannot be used within a virtualized component...". Maybe surround it with flashing lights and "All hope abandon ye who enter here"

Reply:
Updated.